src/libraries/Microsoft.Extensions.DependencyInjection/src/ServiceLookup/CallSiteFactory.cs

```diff
@@ -354,12 +354,19 @@ private ServiceCallSite TryCreateOpenGeneric(ServiceDescriptor descriptor, Type
             bool throwIfCallSiteNotFound)
         {
             var parameterCallSites = new ServiceCallSite[parameters.Length];
-            for (var index = 0; index < parameters.Length; index++)
+            for (int index = 0; index < parameters.Length; index++)
             {
-                var parameterType = parameters[index].ParameterType;
-                var callSite = GetCallSite(parameterType, callSiteChain);
+                Type parameterType = parameters[index].ParameterType;
+                ServiceCallSite callSite = GetCallSite(parameterType, callSiteChain);
 
+
+/* Unmerged change from project 'Microsoft.Extensions.DependencyInjection (net461)'
```

Comment:
Why are we adding these lines?

Reply:
I have no idea where those came from. Thanks. I'll fix it.

Reply:
haha yeah it looked weird... maybe the autofixer put them there because of the multiple configurations?